analyzer/src/funTest/assets/projects/synthetic/pipenv-expected-output.yml

```diff
@@ -162,27 +156,25 @@ packages:
 - id: "PyPI::markupsafe:1.0"
   purl: "pkg:pypi/markupsafe@1.0"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
   declared_licenses:
   - "BSD"
-  - "BSD License"
   declared_licenses_processed:
     spdx_expression: "BSD-3-Clause"
     mapped:
       BSD: "BSD-3-Clause"
-      BSD License: "BSD-3-Clause"
-  description: "Implements a XML/HTML/XHTML Markup safe string for Python"
+  description: "MarkupSafe"
   homepage_url: "http://github.com/pallets/markupsafe"
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/4d/de/32d741db316d8fdb7680822dd37001ef7a448255de9699ab4bfcbdf4172b/MarkupSafe-1.0.tar.gz"
+    url: ""
```

Comment:
Why's this missing?

Reply:
See the response on another similar comment.